### PR TITLE
Fix documentation about building samples

### PR DESCRIPTION
**Pull request by nashif**
Use the cmake variant for some outdated samples/boards/tests.

### CODE REVIEW

**Review by galak — APPROVED** [review]

**Comment by nashif** [comment]
recheck

File: samples/nfc/nfc_hello/README.txt [hunk 1]
```diff
@@ -15,10 +15,14 @@ To test the serial line routines, open a terminal window and type:
 Open another terminal window and type:
 
 For QEMU x86:
-	make pristine && make run
+	mkdir build; cd build
+	cmake -DBOARD=qemu_x86 ..
+	make run
```
Comment (MaureenHelm) [inline samples/nfc/nfc_hello/README.txt]: Why not use zephyr-app-commands?

Comment (nashif) [reply samples/nfc/nfc_hello/README.txt]: this is not a RST file, it is plain text

Comment (MaureenHelm) [reply samples/nfc/nfc_hello/README.txt]: Ok, so why not make it an RST file?

Comment (nashif) [reply samples/nfc/nfc_hello/README.txt]: backlog, but this should be done in some other patch, I just noticed those old calls and figured I fix them, remaining TXT->RST need to be done at some point though.

**Pull request merged**